pkg/endpointmanager/manager.go

```diff
@@ -75,9 +82,14 @@ func NewEndpointManager(epSynchronizer EndpointResourceSynchronizer) *EndpointMa
 	mgr := EndpointManager{
 		endpoints:                    make(map[uint16]*endpoint.Endpoint),
 		endpointsAux:                 make(map[string]*endpoint.Endpoint),
+		mcastManager:                 mcastmanager.New(option.Config.IPv6MCastDevice),
 		EndpointResourceSynchronizer: epSynchronizer,
 	}
 
+	if option.Config.EnableIPv6NDP {
+		mgr.mcastManager = mcastmanager.New(option.Config.IPv6MCastDevice)
+	}
+
```

Comment:
This seems to be duplicated with the configuration of the multicast manager on line 85, or am I missing some detail?